File: webconsole/src/main/resources/_webapp/src/app/charts/model/axis/chrono.axis.ts
import { Axis } from './abstract.axis';

export class ChronoAxis extends Axis {
    public type:string = Axis.CHRONO;
    public unitOfMeasurement:string = "seconds";

    public toJSON():any {
        let _value:any = {};
        _value["@type"] = this.type;
        _value["name"] = this.name;
        return _value;
    }
}

File: webconsole/src/main/java/com/bytex/snamp/web/serviceModel/charts/ChronoAxis.java
package com.bytex.snamp.web.serviceModel.charts;

import org.codehaus.jackson.annotate.JsonIgnore;
import org.codehaus.jackson.annotate.JsonTypeName;

/**
 * @author Roman Sakno
 * @version 2.1
 * @since 2.0
 */
@JsonTypeName("chrono")
public final class ChronoAxis extends Axis {
    public static final String UNIT_OF_MEASUREMENT = "seconds";

    @Override
    @JsonIgnore
    public String getUOM() {
        return UNIT_OF_MEASUREMENT;
    }
}
